script/hardhat/fork/vaultUpgradesVip.ts

```diff
@@ -0,0 +1,429 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import { IAccessControlManager, VAIVault, VRTVault, VRTVaultProxy, XVSVault, XVSVaultProxy } from "../../../typechain";
+import { forking, testVip } from "./vip-framework";
+import { ProposalType } from "./vip-framework/types";
+import { makeProposal } from "./vip-framework/utils";
+import { initMainnetUser } from "./vip-framework/utils";
+
+const ACM = "0x4788629ABc6cFCA10F9f969efdEAa1cF70c23555";
+const XVS_VAULT_PROXY = "0x051100480289e704d20e9DB4804837068f3f9204";
+const VAI_VAULT_PROXY = "0x0667Eed0a0aAb930af74a3dfeDD263A73994f216";
+const VRT_VAULT_PROXY = "0x98bF4786D72AAEF6c714425126Dd92f149e3F334";
+const XVS_OLD = "0xA0c958ca0FfA25253DE0a23f98aD3062F3987073";
+const VAI_OLD = "0x7680C89Eb3e58dEc4D38093B4803be2b7f257360";
+const VRT_OLD = "0xA3EEA5e491AD45caE30F6E0a315A275cc99EE147";
+const XVS_NEW = "0xE6970880279597e806e571ff0A421b99851e8c7e";
+const VAI_NEW = "0x2fA91214f0de7b490591D540Bb12781569Ab754F";
+const VRT_NEW = "0xEf315a89635975996192361D19cDb3EC3c6D633c";
+const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
+const CRITICAL_TIMELOCK = "0x213c446ec11e45b15a6E29C1C1b402B8897f606d";
+const MULTISIG = "0x1C2CAc6ec528c20800B2fe734820D87b581eAA6B";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const LAST_ACCRUING_BLOCK = "27756973";
+
+export const vip105 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-105 Vault Upgrades",
+    description: `
+    Accept the ownership of the XVSVault and VRTVault contracts (previously we had to execute a multisig TX to offer this change)
```

Comment:
We already transfer the ownership of XVSVault in https://app.venus.io/governance/proposal/113, so, only the ownership of the VRTVault should be accepted here